integration/tests/access/cohort1/access_api_test.go

```diff
@@ -1157,3 +1159,176 @@ func (s *AccessAPISuite) TestRejectedInvalidSignatureFormat() {
 		})
 	}
 }
+
+// TestScheduledTransactions tests that scheduled transactions are properly indexed and can be
+// retrieved through the Access API.
+func (s *AccessAPISuite) TestScheduledTransactions() {
+	sc := systemcontracts.SystemContractsForChain(s.net.Root().HeaderBody.ChainID)
+
+	accessClient, err := s.net.ContainerByName("access_2").TestnetClient()
+	s.Require().NoError(err)
+	rpcClient := s.an2Client.RPCClient()
+
+	// Deploy the test contract first
+	txID, err := lib.DeployScheduledCallbackTestContract(accessClient, sc)
```

Comment:
The contract should be deployed by default for a newly bootstrapped network, which we should have here. Here is where it is deployed in execution state bootstrapping: https://github.com/onflow/flow-go/blob/46df27c943a8b5e061e3e2beabb4b5cd56c6959b/fvm/bootstrap.go#L804-L830. (If it isn't deployed by default for some reason, it probably should be.)

We could use the pre-deployed scheduled transaction contract and save a step in the tests here.